platform/src/ConsolePanel.js

```diff
@@ -223,13 +225,18 @@ class ConsolePanel extends Panel {
         editor.hoverLink.on("open", function (e) {
             var location = e.value;
             if (editor.getValue().indexOf(location) > -1) {
+
+            /* TODO Fix link highlighting original epsilon playground behaviour
```

Comment:
It is broken functionality from the initial refactoring of the epsilon playground that assumes a fixed program panel rather than the flexibly defined panels of the education platform. The original behaviour is that any errors shown in the console  are links that when clicked highlight the line of code in the program panel.

Reply:
Thanks for clarifying. Would be good to create an issue for this: it would be good to eventually build similar functionality.

Reply:
Raised https://github.com/mdenet/educationplatform/issues/181